challenge.rb

```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.map { |n| n.capitalize }
```

Comment:
Since it is so common, there is a shorthand for this particular code (calling a method on the block argument):

```ruby
input.map(&:capitalize)
```